Assets/Scripts/PlayerSupervisor.cs

```diff
@@ -84,12 +122,18 @@ public void StartGame()
             return;
         }
 
+        // Only start if the player is ready
+        if (playerState != PlayerState.Ready)
+            return;

```

Comment:
When in a scene where the paddle is managed by a PlayerController, the PlayerState is never set to ready so the ball never launches. My initial instinct is to just add a call for PlayerSupervisor.PlayerReady() in PlayerController.Start(), but PlayerController currently doesn't have any reference to PlayerSupervisor.